server/app/services/applicant/ApplicantData.java

```diff
@@ -149,4 +149,15 @@ public ImmutableMap<Path, String> getFailedUpdates() {
   public boolean updateDidFailAt(Path path) {
     return getFailedUpdates().containsKey(path);
   }
+
+  public Optional<String> getDOB() {
+    String dob = readString(WellKnownPaths.APPLICANT_DOB).get();
+    return Optional.of(dob);
+  }
+
+  public void setDOB(@Nullable String dob) {
+    if (!hasPath(WellKnownPaths.APPLICANT_DOB)) {
+      putString(WellKnownPaths.APPLICANT_DOB, dob);
+    }
+  }
```

Comment:
suggestion (blocking): add unit tests for these methods

Reply:
suggestion (blocking): use CameCase for these method names https://google.github.io/styleguide/javaguide.html#s5.3-camel-case